<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtSquidClass</class>
 <widget class="QMainWindow" name="QtSquidClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtSquid</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>621</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Search">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>611</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="search_VLayout">
       <item>
        <layout class="QHBoxLayout" name="search_HLayout_1">
         <item>
          <widget class="QTableWidget" name="search_resultTable">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="search_VLayout_1_1">
           <item>
            <widget class="QGroupBox" name="search_GBox_1_1_1">
             <layout class="QFormLayout" name="search_FLayout_1_1_1">
              <item row="0" column="1">
               <widget class="QComboBox" name="search_item_availabilityDropdown">
                <item>
                 <property name="text">
                  <string>All</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Free Only</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Used Only</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="search_item_availabilityLabel">
                <property name="text">
                 <string>Type</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QCheckBox" name="search_item_devCheckbox">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="search_item_devLabel">
                <property name="text">
                 <string>Expend</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="search_scanEdit">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>133</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="search_scanBtn">
             <property name="text">
              <string>Scan</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="search_HLayout_2">
         <item>
          <widget class="QPushButton" name="search_runBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="search_outputLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Edit">
     <attribute name="title">
      <string>Edit</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>9</y>
        <width>611</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="edit_VLayout">
       <item>
        <layout class="QHBoxLayout" name="edit_HLayout_1">
         <item>
          <widget class="QPushButton" name="edit_addBtn">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="edit_removeBtn">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="edit_FLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="edit_form_barcodeLabel">
           <property name="text">
            <string>Barcode</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="edit_form_barcodeEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="edit_form_qtySpinbox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="edit_form_qtyLabel">
           <property name="text">
            <string>Quantity</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="edit_HLayout_2">
         <item>
          <widget class="QPushButton" name="edit_runBtn">
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="edit_outputLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Connection">
     <attribute name="title">
      <string>Connection</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>9</y>
        <width>611</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="conn_VLayout">
       <item>
        <layout class="QHBoxLayout" name="conn_HLayout_1">
         <item>
          <layout class="QFormLayout" name="conn_FLayout_1_1">
           <item row="1" column="0">
            <widget class="QLabel" name="conn_labelAddress">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="text">
              <string>Address</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="conn_editAddress">
             <property name="maxLength">
              <number>32</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="conn_labelPort">
             <property name="text">
              <string>Port</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="conn_editPort">
             <property name="contextMenuPolicy">
              <enum>Qt::NoContextMenu</enum>
             </property>
             <property name="maxLength">
              <number>4</number>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="conn_btnSaveDatabase">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="conn_FLayout_1_2">
           <item row="2" column="0">
            <widget class="QLabel" name="conn_labelPassword">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="conn_editPassword">
             <property name="maxLength">
              <number>128</number>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="conn_btnSaveUser">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="conn_editUsername">
             <property name="maxLength">
              <number>128</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="conn_labelUsername">
             <property name="text">
              <string>Username</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="conn_HLayout_2">
         <item>
          <widget class="QPushButton" name="conn_runBtn">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="conn_outputLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Permission">
     <attribute name="title">
      <string>Permission</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>9</y>
        <width>611</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="perm_VLayout">
       <item>
        <layout class="QHBoxLayout" name="perm_HLayout_1">
         <item>
          <widget class="QTreeView" name="perm_permTree"/>
         </item>
         <item>
          <layout class="QFormLayout" name="perm_FLayout_1_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="perm_HLayout_2">
         <item>
          <widget class="QPushButton" name="perm_refreshBtn">
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="perm_outputLabel">
           <property name="text">
            <string>text</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>637</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOutils">
    <property name="title">
     <string>Outils</string>
    </property>
    <widget class="QMenu" name="menuOptions">
     <property name="title">
      <string>Options</string>
     </property>
     <widget class="QMenu" name="menuDatabase">
      <property name="title">
       <string>Database</string>
      </property>
      <addaction name="actionSilentConnect"/>
     </widget>
     <addaction name="actionStyleSheet"/>
     <addaction name="menuDatabase"/>
    </widget>
    <addaction name="menuOptions"/>
   </widget>
   <widget class="QMenu" name="menuRaccourcis">
    <property name="title">
     <string>Raccourcis</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <widget class="QMenu" name="menuLinks">
     <property name="title">
      <string>Links</string>
     </property>
     <addaction name="actionLink_StyleSheet"/>
     <addaction name="actionLink_Email"/>
    </widget>
    <addaction name="menuLinks"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuOutils"/>
   <addaction name="menuRaccourcis"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStyle">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionStyleSheet">
   <property name="text">
    <string>Style</string>
   </property>
  </action>
  <action name="actionSilentConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Silent Connection</string>
   </property>
  </action>
  <action name="actionLink_StyleSheet">
   <property name="text">
    <string>Stylesheet</string>
   </property>
  </action>
  <action name="actionLink_Email">
   <property name="text">
    <string>Email</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtSquid.qrc"/>
 </resources>
 <connections/>
</ui>
